tools/git-clang-tidy

```diff
@@ -0,0 +1,597 @@
+#!/usr/bin/env python
```

Comment:
Do we really need our own copy of this file?
This means additional maintenance effort if there is e.g. a newer version with fixes...

Reply:
I actually modified it to run clang-tidy instead of clang-format.
Was just a try if this works better.